tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/CMakeLists.txt

```diff
@@ -0,0 +1,65 @@
+cmake_minimum_required(VERSION 3.13.1)
+find_package(Zephyr)
+project(tf_lite_micro_speech)
+
+set(TF_MICRO_DIR ${PROJECT_SOURCE_DIR}/../../../)
+file(GLOB MICRO_SOURCES ${TF_MICRO_DIR}/*cc)
```

Comment:
It's good to know that, but is that a big problem in our project? I think we can leave it as it is, code is more readable and according to docs you linked it's not unstable in any way, just have to know the cons. Are we going to modify those files anyway?

Reply:
I'm just pointing this out now, to highlight possible error prone areas. I don't know if this will be a problem in our case. I think we should leave this as it is for now and refactor in the future if necessary.